README.md

```diff
@@ -1,56 +1,138 @@
+Then you can either run `node_modules/.bin/fractal start` to start up the project, or create an alias under the `scripts` section in your package.json as a shortcut.
 
-## Requirements
+e.g.
 
-Any LTS (Long-term stable) version of NodeJS. At the time of writing this is 6 (Boron) and 8 (Carbon).
+```json
+scripts: {
+    "fractal:start": "fractal start --sync",
+    "fractal:build": "fractal build"
+}
+```
 
-If you're on an earlier (unsupported) version of NodeJS, then see the below compatibility table.
+then
 
-| NodeJS version | Fractal version |
-| -------------- | --------------- |
-| 4.x            | 1.1.7           |
-| 5.x            | 1.1.7           |
+```shell
+npm run fractal:start
+```
 
-## Testing
+### Installing globally
 
-Fractal is a project that has recently evolved rapidly and organically from a proof-of-concept prototype into a more stable, mature tool. Because of this it's currently pretty far behind where it should be in terms of test coverage. Any contributions on this front would be most welcome!
+```shell
+npm i -g @frctl/fractal
+```
 
-Existing tests can be run using the `npm test` command.
+This will also give you global access to the `fractal` command which you can use to scaffold a new Fractal project with `fractal new`.
+
+The downside is that it's then difficult to use different Fractal versions on different projects.
+
+This option is not recommended until a global Fractal install is capable of offloading to a project specific version.
+
+## Examples
+
+While there are no official examples or demo instances, there are lots of public examples on the [Awesome Fractal](https://github.com/frctl/awesome-fractal) repo.
 
 ## Contributing
 
 Fractal has an active group of contributors but we are always looking for more help. If you are interested in contributing then please come and say hi on [Fractal's Discord server](https://discord.gg/vuRz4Yx) - or of course we will always welcome pull requests on any of the [frctl organisation](https://github.com/frctl) repos.
 
 Please note we have a [code of conduct](.github/CODE_OF_CONDUCT.md), please follow it in all your interactions with the project.
 
-## Credits
+### Testing
+
+Fractal is a project that has recently evolved rapidly and organically from a proof-of-concept prototype into a more stable, mature tool. Because of this it's currently pretty far behind where it should be in terms of test coverage. Any contributions on this front would be most welcome!
+
+Existing tests can be run using the `npm test` command.
+
+## Contributors ✨
+
+Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/docs/en/emoji-key)):
+
+<!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-disable -->
+<table>
```

Comment:
can we make this table a bit more balanced visually? it renders a bit weird to my eye.

![image](https://user-images.githubusercontent.com/1892091/78972998-a8856480-7b17-11ea-90d0-d3dc87e468d3.png)

Reply:
There's not a huge amount we can do, as it's auto-generated by the auto-contributors CLI.

I've switched it to 6 columns in the config, so it's a little more balanced. We're kind of limited by tables though, so there's not a lot else we can do.